scripts/variantstore/scripts/create_vt_bqloadjson_from_annotations.py

```diff
@@ -61,6 +65,17 @@
     "clinvar_phenotype": "phenotypes" # nullable
 }
 
+vat_clinvar_review_status_dictionary = { # https://www.ncbi.nlm.nih.gov/clinvar/docs/review_status/
+    "no classification for the individual variant": "none",
+    "no classification provided": "none",
+    "no assertion criteria provided": "none",
```

Comment:
oh I wish the Clinvar folks had chosen "zero" to express this concept rather than "none"...